### PR TITLE
Escape pod shuttlification

### PR DESCRIPTION
**Pull request by MadmanMartian**
Shuttlifiy escape pods, rather than having them be an area that teleports around in their own weird way.

Tested. Pods work through the shuttle panel and during natural escape shuttle movement.

### CODE REVIEW

**Comment by MadmanMartian** [comment]
give me my map render, bot!

File: code/controllers/shuttle_controller.dm [hunk 1]
```diff
@@ -126,14 +120,32 @@ datum/emergency_shuttle/proc/setdirection(var/dirn)
...
+			if(!S.move_to_dock(S.transit_port, 0))
+				message_admins("Warning: [S] failed to move to transit.")
+	spawn()
+		for(var/obj/machinery/door/D in S.linked_area)
+			if(destination == "transit")
+				D.close()
+			else
+				D.open()
+
+	/*
```
Comment (DamianX) [inline code/controllers/shuttle_controller.dm]: delete this whole block?

**Review by DamianX — COMMENTED** [review]
>using `global` for global variables
>`proc/` instead of `/proc/`
absolutely disgusting

**Comment by DamianX** [comment]
oh and let's not kill this with conflicts

**Comment by MadmanMartian** [comment]
woo!

**Comment by MadmanMartian** [comment]
Escaping via escape pod will need reworking. Now I remember why I needed that is_on_shuttle check

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -297,19 +297,20 @@ var/list/potential_theft_objectives=list(
...
+	for(var/mob/living/player in escape_shuttle.get_occupants(TRUE))
+		if(is_type_in_list(player, protected_mobs)) //These boys are okay to be on the shuttle
 			continue
-		if (player.mind && (player.mind != owner))
-			if(player.stat != DEAD)			//they're not dead!
-				if(get_turf(player) in shuttle)
-					return 0
+		if(player == owner.current) //We are this boy
+			continue
+		if(player.stat == 2) //They are dead
```
Comment (DamianX) [inline code/game/gamemodes/objective.dm]: `2` -> `DEAD`

File: code/game/gamemodes/objective.dm [hunk 3]
```diff
@@ -324,15 +325,15 @@ var/list/potential_theft_objectives=list(
...
-		if(player.type in protected_mobs)
+	var/list/protected_mobs = list(/mob/living/silicon/ai, /mob/living/silicon/pai, /mob/living/silicon/robot, /mob/living/silicon/robot/mommi, /mob/living/simple_animal/borer)
+	for(var/mob/living/player in escape_shuttle.get_occupants(TRUE))
+		if(is_type_in_list(player, protected_mobs)) //These boys are okay to be on the shuttle
 			continue
-		if (player.mind)
-			if (player.stat != 2)
-				if (get_turf(player) in shuttle)
-					return 0
+		if(player.stat == 2) //Do not suffer the human to live
```
Comment (DamianX) [inline code/game/gamemodes/objective.dm]: `2` -> `DEAD`

**Comment by Kurfursten** [comment]
Finish this so I can get packed priority

File: code/game/shuttles/emergency.dm [hunk 4]
```diff
@@ -39,10 +39,121 @@ var/global/datum/shuttle/escape/escape_shuttle = new(starting_area=/area/shuttle
...
+
+var/global/datum/shuttle/escape/pod/two/EP2 = new(starting_area=/area/shuttle/escape_pod2)
+
+var/global/datum/shuttle/escape/pod/three/EP3 = new(starting_area=/area/shuttle/escape_pod3)
+
+var/global/datum/shuttle/escape/pod/four/EP4 = new(starting_area=/area/shuttle/escape_pod4)
+
+var/global/datum/shuttle/escape/pod/five/EP5 = new(starting_area=/area/shuttle/escape_pod5)
+
+/datum/shuttle/escae/pod/one/name = "Escape pod 1"
```
Comment (DamianX) [inline code/game/shuttles/emergency.dm]: escae?

File: code/game/shuttles/emergency.dm [hunk 5]
```diff
@@ -39,10 +39,121 @@ var/global/datum/shuttle/escape/escape_shuttle = new(starting_area=/area/shuttle
...
+/obj/docking_port/destination/pod1/centcom
+	areaname = "central command"
+
+/obj/docking_port/destination/pod1/station
+	areaname = "station dock"
+
+/obj/docking_port/destination/pod1/transit
+	areaname = "hyperspace (pod 1)"
+
+/datum/shuttle/escae/pod/two/name = "Escape pod 2"
```
Comment (DamianX) [inline code/game/shuttles/emergency.dm]: escae?

File: code/game/shuttles/emergency.dm [hunk 6]
```diff
@@ -39,10 +39,121 @@ var/global/datum/shuttle/escape/escape_shuttle = new(starting_area=/area/shuttle
...
+	areaname = "central command"
+
+/obj/docking_port/destination/pod2/station
+	areaname = "station dock"
+
+/obj/docking_port/destination/pod2/transit
+	areaname = "hyperspace (pod 2)"
+
+
+/datum/shuttle/escae/pod/three/name = "Escape pod 3"
```
Comment (DamianX) [inline code/game/shuttles/emergency.dm]: escae?

File: code/game/shuttles/emergency.dm [hunk 7]
```diff
@@ -39,10 +39,121 @@ var/global/datum/shuttle/escape/escape_shuttle = new(starting_area=/area/shuttle
...
+/obj/docking_port/destination/pod3/centcom
+	areaname = "central command"
+
+/obj/docking_port/destination/pod3/station
+	areaname = "station dock"
+
+/obj/docking_port/destination/pod3/transit
+	areaname = "hyperspace (pod 3)"
+
+/datum/shuttle/escae/pod/four/name = "Escape pod 4"
```
Comment (DamianX) [inline code/game/shuttles/emergency.dm]: escae?

File: code/game/shuttles/emergency.dm [hunk 8]
```diff
@@ -39,10 +39,121 @@ var/global/datum/shuttle/escape/escape_shuttle = new(starting_area=/area/shuttle
...
+/obj/docking_port/destination/pod4/centcom
+	areaname = "central command"
+
+/obj/docking_port/destination/pod4/station
+	areaname = "station dock"
+
+/obj/docking_port/destination/pod4/transit
+	areaname = "hyperspace (pod 4)"
+
+/datum/shuttle/escae/pod/five/name = "Escape pod 5"
```
Comment (DamianX) [inline code/game/shuttles/emergency.dm]: escae?

**Review by DamianX — CHANGES_REQUESTED** [review]

**Pull request merged**